dateutil/parser.py

```diff
@@ -1066,15 +1063,15 @@ def _parse(self, timestr, dayfirst=None, yearfirst=None, fuzzy=False,
                         res.month, res.day, res.year = ymd
 
         except (IndexError, ValueError, AssertionError):
-            return None
+            return None, None
 
         if not info.validate(res):
-            return None
+            return None, None
 
         if fuzzy_with_tokens:
             return res, tuple(skipped_tokens)
         else:
-            return res
+            return res, None
```

Comment:
In my opinion, because return will always be returning a tuple - it would probably be best if the internal _parse method always return the skipped_tokens. These can then just be ignored if required.